### PR TITLE
fix: US132386 allow user to create new grade when attempts present.

### PR DESCRIPTION
**Pull request by mdulat**
https://rally1.rallydev.com/#/?detail=/userstory/610526922887&fdp=true

After attempts submitted, user should be able to attach a new grade and change its Grade Out Of, even after save in place.
Only existing grade's Grade Out Of should not be editable.

### CODE REVIEW

**Comment by mostlyfabulous** [comment]
So if attempts are submitted, once they:
- attach a new grade, 
- save in place, 
- refresh the page,
they will not be able to edit the grade out of right?

File: components/d2l-activity-editor/d2l-activity-score-editor.js [hunk 1]
```diff
@@ -546,6 +562,7 @@ class ActivityScoreEditor extends ActivityEditorMixin(SkeletonMixin(LocalizeActi
 			scoring.setGradeMaxPoints('');
 		}
 		scoring.setCanUpdateScoring(scoring._entity.canUpdateScoring());
+		this._saveInPlaceAfterNewGrade = false;
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-score-editor.js:565]: Say the user is attaching a new grade (attempts already submitted), they save in place, the update the grade out of, will this now set `canEditScoreOutOf ` to `false`?

**Comment by mostlyfabulous** [comment]
Visual diffs have bigger image dimensions, looks legit to merge:

https://visual-diff.d2l.dev/screenshots/BrightspaceHypermediaComponents/activities/d2l-quick-eval-widget/2022-04-08-02.06.12.493-a83428e4a50cf228d1bacf16653682f27b96865a-report.html

**Comment by mdulat** [comment]
> So if attempts are submitted, once they:
> 
> * attach a new grade,
> * save in place,
> * refresh the page,
>   they will not be able to edit the grade out of right?

That's correct. Once the page is refreshed, or closed and re-opened it's now an existing grade and they won't be able to edit the grade out of.

File: components/d2l-activity-editor/d2l-activity-score-editor.js (hunk 1 above)
Comment (mdulat) [reply components/d2l-activity-editor/d2l-activity-score-editor.js:565]: No it won't. The above line only sets it to false when they change the gradebook status to "not in gradebook".

**Review by dustxd — APPROVED** [review]

**Comment by Scy-D2L** [comment]
> Visual diffs have bigger image dimensions, looks legit to merge:
> 
> https://visual-diff.d2l.dev/screenshots/BrightspaceHypermediaComponents/activities/d2l-quick-eval-widget/2022-04-08-02.06.12.493-a83428e4a50cf228d1bacf16653682f27b96865a-report.html

The spacing between the items has changed but that should be related to changes made to the list components recently, it was [called out in Slack](https://d2l.slack.com/archives/C0PHG3QB0/p1649355370086289) that they might alter vdiffs.

**Review by Scy-D2L — APPROVED** [review]

**Pull request merged**